react.json

```diff
@@ -75,7 +75,16 @@
         "react/style-prop-object": "error",
         "react-hooks/exhaustive-deps": "error",
         "react-hooks/rules-of-hooks": "error",
-        "import/dynamic-import-chunkname": "error"
+        "import/dynamic-import-chunkname": "error",
+        "import/no-extraneous-dependencies": [
```

Comment:
I think this rule is to prevent people from importing modules that are not listed in `package.json`. How come it affects spec files? Also, does this override the default?

Reply:
> Also, does this override the default?

It only overrides the default `devDependencies` globs, so it doesn't override the rest.

>How come it affects spec files?

The default glob for `devDependencies` is set to `["**/*.test.js", "**/*.spec.js"]`. We have a naming convention for files in `ng-add-edit-products`, but only for files in the `app-next` folder since they're only React components. We'd like to start using this naming convention in other apps going forward, but the default glob for `devDependencies` affects files that we name like so: `src/app-next/redux/availableChannels/spec.ts`. So this should cover both old and new naming conventions.

Reply:
Right I see, you've configured Webpack to resolve files using the base path of the project rather than using relative paths. That's why the linter thinks that you're importing a NPM dependency rather than a file local to your project. And this list just tells the linter to not lint these files. Is that really what you want? What happens if you try to import a package that's not listed in `package.json` in these files?

Reply:
Oh and have you looked into https://github.com/benmosher/eslint-plugin-import#resolvers

Reply:
Hi @chanceaclark! This rule is `off` in this eslint configuration. I'm not sure why we are adding it here. Also, I could not find anywhere that the default glob for `devDependencies` is `["**/*.test.js", "**/*.spec.js"]`, is there a piece of documentation you could point out to support that?

Where you trying to change this in the `ng-build` eslint config?